vmm/scripts/image/centos/build_rootfs.sh

```diff
@@ -15,40 +15,50 @@
 
 set -e
 
+golang_version="1.20.5"
+
 make_vmm_task() {
     local repo_dir="$1"
 
     # install cmake3 because some rust modules depends on it
     # centos 7 install cmake 2.8 by default, has to add epel repo to install cmake3
     . /etc/os-release
     if [ ${VERSION_ID} -le 7 ]; then
-	      yum install -y epel-release
-	      yum install -y cmake3 make gcc gcc-c++
-	      rm -f /usr/bin/cmake
-	      ln -s /usr/bin/cmake3 /usr/bin/cmake
+        yum install -y epel-release
+        yum install -y cmake3 make gcc gcc-c++ wget
+        rm -f /usr/bin/cmake
+        ln -s /usr/bin/cmake3 /usr/bin/cmake
     else
-	      # CentOS Linux 8 had reached the End Of Life (EOL) on December 31st, 2021. It means that CentOS 8 will no longer receive development resources from the official CentOS project. After Dec 31st, 2021, if you need to update your CentOS, you need to change the mirrors to vault.centos.org where they will be archived permanently. Alternatively, you may want to upgrade to CentOS Stream.
-	      sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
-	      sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
-	      yum clean all
-	      yum install cmake make gcc-c++
+        # CentOS Linux 8 had reached the End Of Life (EOL) on December 31st, 2021. It means that CentOS 8 will no longer receive development resources from the official CentOS project. After Dec 31st, 2021, if you need to update your CentOS, you need to change the mirrors to vault.centos.org where they will be archived permanently. Alternatively, you may want to upgrade to CentOS Stream.
+        sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+        sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
+        yum clean all
+        yum install cmake make gcc-c++ wget
     fi
 
     # install rust to compile vmm-task
     pushd ${repo_dir}
-    ARCH=`uname -m`
+    ARCH=$(uname -m)
 
     source vmm/scripts/image/install_rust.sh
 
     make bin/vmm-task ARCH=${ARCH}
     popd
 }
 
+install_golang() {
+    pushd /home/
+    rm -f go${golang_version}.linux-arm64.tar.gz
+    wget "https://go.dev/dl/go${golang_version}.linux-arm64.tar.gz"
+    rm -rf /usr/local/go && tar -C /usr/local -xzf "go${golang_version}.linux-arm64.tar.gz"
+    echo "export PATH=$PATH:/usr/local/go/bin" >>/etc/profile
+    source /etc/profile
+    go version
```

Comment:
need execute permission

Reply:
The `go` binary extracts from the go${golang_version}.linux-${arch}.tar.gz keeps the execution permission as default, so we don't need to change it.